exporter/splunkhecexporter/README.md

```diff
@@ -22,6 +22,14 @@ The following configuration options can also be configured:
 - `disable_compression` (default: false): Whether to disable gzip compression over HTTP.
 - `timeout` (default: 10s): HTTP timeout when sending data.
 - `insecure_skip_verify` (default: false): Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS.
+- `insecure` (default = false): When set to true disables verifying the server's certificate chain and host name. The
```

Comment:
I don't think insecure does what it says. I'll have to look into it separately. It's mainly for gRPC, for HTTP whether TLS is used is controlled by whether scheme is http/https. I think we can just delete documentation for insecure here.

Reply:
yeah @jrcamp thats cool :)